io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,157 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "23.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=fallback-x11
+- --share=ipc
+- --share=network
+- --socket=pulseaudio
+- --filesystem=~/.var/app:ro
+- --filesystem=home
```

Comment:
What is home needed for?

Reply:
To both read the game files from and to write the save files (Kept the standard path in order also to allow the user to backup their save files)

Reply:
Ok, so I made the home folder read only and add special rights to write ONLY on .doombfa.
I hope now is better